src/lib.ts

```diff
@@ -316,15 +310,16 @@ export function printHelpMenu(): void {
 export function pillkeeperOpportunityCost(): number {
   const canTreasury = canAdventure($location`Cobb's Knob Treasury`);
 
-  const alternateUse = [
-    { can: canTreasury, value: 3 * get("valueOfAdventure") },
-    {
-      can: realmAvailable("sleaze"),
-      value: 40000,
-    },
-  ]
-    .filter((x) => x.can)
-    .sort((a, b) => b.value - a.value)[0];
+  const alternateUse = maxBy(
+    [
+      { can: canTreasury, value: 3 * get("valueOfAdventure") },
```

Comment:
okay i have a thought here. we store embezzlers as 3x VOA in a lot of different places. what if we start changing that to valueOfEmbezzler upwards in lib.ts, make that equal to 3x VOA for now, swap it out in the various locations it's done, and someday have better evaluation procedures? this way if we ever do a better job evaluating it we only have to do it in one spot.

Reply:
made separate issue; resolving convo

Reply:
Uhoh this one could crash in cs